packages/react-component-library/src/components/List/List.test.tsx

```diff
@@ -207,4 +207,23 @@ describe('List', () => {
       )
     })
   })
+
+  describe('when a list item has multiple children ', () => {
+    beforeEach(() => {
+      const extraItemText = 'extra text'
+      wrapper = render(
+        <List>
+          <ListItem onClick={jest.fn()} title="List item">
+            Main text: {extraItemText}
+          </ListItem>
+        </List>
+      )
+    })
+
+    it("renders the item's text", () => {
```

Comment:
These should be single quotes.

```suggestion
    it('renders the item's text', () => {
```

Reply:
They're double due to the embedded single quote – unless you're suggesting escaping instead?

Reply:
Ah sorry I didn't spot that - i'd probably escape it but doesn't really matter either way.

I've approved it 👍